### PR TITLE
- adapted to changes in snapper

### PR DESCRIPTION
**Pull request by aschnell**

### CODE REVIEW

**Comment by aschnell** [comment]
Failure in Travis is due to compilation with old libsnapper.

File: agent-snapper/src/SnapperAgent.cc [hunk 1]
```diff
@@ -520,21 +518,22 @@ YCPValue SnapperAgent::Execute(const YCPPath &path, const YCPValue& arg,
...
-            string cleanup      = getValue (argmap, YCPString ("cleanup"), "");
-            string type         = getValue (argmap, YCPString ("type"), "single");
-            YCPMap userdata     = getMapValue (argmap, YCPString ("userdata"));
+	    string type = getValue (argmap, YCPString ("type"), "single");
+
+	    SCD scd;
+	    scd.uid = getuid();
+	    scd.description = getValue(argmap, YCPString("description"), "");
+	    scd.cleanup = getValue(argmap, YCPString("cleanup"), "");
+	    scd.userdata = ycpmap2stringmap(getMapValue(argmap, YCPString("userdata")));
```
Comment (jreidinger) [inline agent-snapper/src/SnapperAgent.cc]: I am not sure if it is problem of github or mixture of tabs and space, but it looks like inconsistent indentation.

**Comment by lslezak** [comment]
The problem is that the new libsnapper uses some new C++11 features not present in the outdated Travis gcc... :worried: 

So ignore the failures for now.

File: agent-snapper/src/SnapperAgent.cc (hunk 1 above)
Comment (aschnell) [inline agent-snapper/src/SnapperAgent.cc]: AFAIS old code didn't use tabs.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**